packages/block-editor/src/components/grid/grid-item-resizer.js

```diff
@@ -119,28 +119,22 @@ function GridItemResizerInner( {
 				} }
 				bounds={ bounds }
 				boundsByDirection
+				onPointerDown={ ( { target, pointerId } ) => {
+					/*
+					 * Captures the pointer to avoid hiccups while dragging over objects
+					 * like iframes and ensures that the event to end the drag is
+					 * captured by the target (resize handle) whether or not it’s under
+					 * the pointer.
+					 */
+					target.setPointerCapture( pointerId );
+				} }
```

Comment:
I think it’d nice to put this in `ResizableBox` as it should be okay for any use case.
<details><summary>I gave it a quick try but typing it perfectly is an exercise I didn’t complete (had to use `@ts-ignore`).</summary>

```diff
diff --git a/packages/components/src/resizable-box/index.tsx b/packages/components/src/resizable-box/index.tsx
index 1b05270ea0..96c3044b7b 100644
--- a/packages/components/src/resizable-box/index.tsx
+++ b/packages/components/src/resizable-box/index.tsx
@@ -9,7 +9,7 @@ import { forwardRef } from '@wordpress/element';
 import clsx from 'clsx';
 import { Resizable } from 're-resizable';
 import type { ResizableProps } from 're-resizable';
-import type { ReactNode, ForwardedRef } from 'react';
+import type { ReactNode, ForwardedRef, PointerEventHandler } from 'react';
 
 /**
  * Internal dependencies
@@ -92,6 +92,7 @@ type ResizableBoxProps = ResizableProps & {
 	showHandle?: boolean;
 	__experimentalShowTooltip?: boolean;
 	__experimentalTooltipProps?: Parameters< typeof ResizeTooltip >[ 0 ];
+	onPointerDown: PointerEventHandler;
 };
 
 function UnforwardedResizableBox(
@@ -116,6 +117,17 @@ function UnforwardedResizableBox(
 			handleStyles={ HANDLE_STYLES }
 			ref={ ref }
 			{ ...props }
+			// @ts-ignore -- Ignore that ResizableProps does not include `onPointerDown`
+			onPointerDown={ ( event ) => {
+				/*
+				 * Captures the pointer to avoid hiccups while dragging over objects
+				 * like iframes and ensures that the event to end the drag is
+				 * captured by the target (resize handle) whether or not it’s under
+				 * the pointer.
+				 */
+				event.target.setPointerCapture( event.pointerId );
+				props.onPointerDown( event );
+			} }
 		>
 			{ children }
 			{ showTooltip && <ResizeTooltip { ...tooltipProps } /> }

```
</details>

Reply:
Ideally this would be fixed in `re-resizable` but you could do something like this as a workaround:

```diff
diff --git a/packages/components/src/resizable-box/index.tsx b/packages/components/src/resizable-box/index.tsx
index 1b05270ea0b..28d3a7cea74 100644
--- a/packages/components/src/resizable-box/index.tsx
+++ b/packages/components/src/resizable-box/index.tsx
@@ -9,7 +9,7 @@ import { forwardRef } from '@wordpress/element';
 import clsx from 'clsx';
 import { Resizable } from 're-resizable';
 import type { ResizableProps } from 're-resizable';
-import type { ReactNode, ForwardedRef } from 'react';
+import type { ReactNode, ForwardedRef, PointerEventHandler } from 'react';
 
 /**
  * Internal dependencies
@@ -87,11 +87,19 @@ const HANDLE_STYLES = {
 	bottomLeft: HANDLE_STYLES_OVERRIDES,
 };
 
+declare module 're-resizable' {
+	// eslint-disable-next-line @typescript-eslint/no-shadow
+	interface ResizableProps {
+		onPointerDown: PointerEventHandler;
+	}
+}
+
 type ResizableBoxProps = ResizableProps & {
 	children: ReactNode;
 	showHandle?: boolean;
 	__experimentalShowTooltip?: boolean;
 	__experimentalTooltipProps?: Parameters< typeof ResizeTooltip >[ 0 ];
+	onPointerDown: PointerEventHandler;
 };
 
 function UnforwardedResizableBox(
@@ -116,6 +124,11 @@ function UnforwardedResizableBox(
 			handleStyles={ HANDLE_STYLES }
 			ref={ ref }
 			{ ...props }
+			onPointerDown={ ( event ) => {
+				const target = event.target as Element;
+				target.setPointerCapture( event.pointerId );
+				props.onPointerDown( event );
+			} }
 		>
 			{ children }
 			{ showTooltip && <ResizeTooltip { ...tooltipProps } /> }
```

Reply:
Thanks for the tip!